titan/params/haart.yml

```diff
@@ -0,0 +1,22 @@
+haart:
+  use_cap:
+    default: false
+    description: Whether "prob" defined in demographics.race.sex_type.haart is a percent of diagnosed, HIV+ agents instead of a per-time-step probability for HAART coverage
+    type: boolean
+  reinit:
```

Comment:
should this be `use_reinit` or similar? we're also using reinit as the prob in demographics